config.py

```diff
@@ -21,6 +21,9 @@
 if not os.environ.get('COBOT_PREFIX'):
     BOT_ALT_PREFIXES = ('cobot ', )
 
+BOT_DEPRECATED_PREFIXES = os.environ.get(
+    'BOT_DEPRECATED_PREFIXES', '').split() or ('cobot', )
```

Comment:
The prefix should be `cobot ` .  The default is `!` , and it is possible that people might want to use this plugin to deprecate `!`, without any spaces being added after it.

Reply:
I dont understand this

> The prefix should be cobot.

Do you mean we should hardcode it instead of getting from env var?

Reply:
Oh, I am sorry ... github isnt rendering that correctly.  `'cobot' + ' '`